__tests__/app.test.js

```diff
@@ -91,3 +90,56 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/", () => {
+  test("200: Responds with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+          expect(typeof article.comment_count).toBe("number");
+          /*
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+          */
+          expect(article).not.toMatchObject({
+            body: expect.any(String),
+          });
+          if (article.article_id === 1) {
+            expect(article.comment_count).toBe(11);
+          }
```

Comment:
I'd say the above is not super necessary because it only proves the exact number in one article, and doing this for every article would be a pain. I think the assertion above where you are checking that every article has a comment count is enough personally - but it's not necessarily wrong that you have this check in there too